runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -663,6 +663,14 @@ gcParseXlpOption(J9JavaVM *vm)
 			j9nls_printf(PORTLIB, J9NLS_INFO, J9NLS_GC_OPTIONS_XLP_EXTRA_COMMA);
 		}
 	}
+	else if (vm->vmRuntimeStateListener.largePageSizeInBytes != NULL) {
```

Comment:
Why is the `vmRuntimeStateListener` being used to pass data here?

Reply:
Even you do use `vm->vmRuntimeStateListener.largePageSizeInBytes` where it is initialized? I can not see explicit initialization in this PR. Is it hidden somewhere inside mapping macros?

Reply:
When I suggested to use `vmRuntimeStateListener` I imagined:
- we parse `-XX:LargePageSizeInBytes=<size>` option in VM code (in assumption we want parse all -XX options in one place) and get numeric value (so handle all parsing errors here)
- use `vm->vmRuntimeStateListener.largePageSizeInBytes` to pass this value to GC and JIT code symmetrically
- handle such value in proper places in GC and JIT code as a "parsed" with context control
If we are not firm on parsing XX options in one place we can handle `-XX:LargePageSizeInBytes=<size>` independently in GC and JIT

Reply:
Why in `vmRuntimeStateListener`?  The large page size has nothing to do with that struct

Reply:
Well I agree. However we do have examples how this structure is used to pass XX parsed information to other components, for example for `GC_ON_IDLE`.

Reply:
That makes sense though as the `vmRuntimeStateListener` is related to the idle detection and memory giveback feature.

Reply:
Stale code. Resolving.